pkg/kubelet/cm/cpumanager/policy_static.go

```diff
@@ -201,16 +201,17 @@ func (p *staticPolicy) validateState(s state.State) error {
 		}
 		// state is empty initialize
 		allCPUs := p.topology.CPUDetails.CPUs()
-		s.SetDefaultCPUSet(allCPUs)
+		nonReservedCPUs := allCPUs.Difference(p.reservedCPUs)
+		s.SetDefaultCPUSet(nonReservedCPUs)
```

Comment:
after a quick pass, I'd extract a helper like
```
func (p *staticPolicy) allocatableCPUs() cpuset.CPUSet {
	return p.topology.CPUDetails.CPUs().Difference(p.reservedCPUs)
}
```
then in the calling site above I'd use
```
	// state is empty initialize
       s.SetDefaultCPUSet(p.allocatableCPUs())
       return nil
```

and then below
```
// GetAllocatableCPUs returns the total set of CPUs available for allocation.
func (p *staticPolicy) GetAllocatableCPUs(s state.State) cpuset.CPUSet {
	return p.allocatableCPUs()
}
```

Still need to check `GetAvailableCPUs()`